content/en/docs/how-tos/cluster-claim.md

```diff
@@ -214,9 +214,35 @@ document.
 
 The cluster pools are maintained by Hive behind the scenes, so installation failures, cloud platform account
 misconfigurations and similar issues are not exposed to actual CI jobs: the jobs will simply never successfully claim a
-cluster if Hive fails to install them. Currently, cluster pool owners cannot access the logs or other troubleshooting
-information on the `hive` cluster. For now, work with DPTP (ping `@dptp-helpdesk` on #forum-testplatform Slack channel)
-to troubleshoot pools that never get ready clusters.
+cluster if Hive fails to install them.
+
+The `ClusterRole` `hive-cluster-pool-admin` will [contains the permissions](https://github.com/openshift/hive/blob/master/docs/clusterpools.md#managing-admins-for-cluster-pools) to all resources created for a pool.
```

Comment:
I'd split this in two parts:

1) Add a section to the "add a new pool" procedure where you instruct new pool conributors to add the RBAC manifest, with a very brief eplanation that this will allow cluster pool owners to troubleshoot/inspect how Hive installs clusters in their pool.
2) Add a paragraph here that describes how to actually troubleshoot a pool, provided you are subject to the RBAC added in 1)

Reply:
Updated.